packages/core/src/components/chip/chip-mixin.scss

```diff
@@ -109,6 +109,15 @@ $border-radius: 100px;
   .container.outline {
       border-width: 1px;
       border-style: solid;
+
+      .icon {
+        padding-right: calc(0.75rem - 1px);
+        padding-left: calc(0.75rem - 1px);
```

Comment:
It looks like this overwrites the reduced margin-left from the close icon variant (see vrt snapshot)